kbase-extension/static/kbase/js/widgets/appWidgets2/input/selectInput.js

```diff
@@ -75,103 +66,77 @@ define([
         // DOM EVENTS
 
         function handleChanged() {
-            return {
-                type: 'change',
-                handler: function () {
-                    importControlValue()
-                        .then((value) => {
-                            model.value = value;
-                            channel.emit('changed', {
-                                newValue: value,
+            importControlValue()
+                .then((value) => {
+                    model.value = value;
+                    channel.emit('changed', {
+                        newValue: value,
+                    });
+                    return validate(value);
+                })
+                .then((result) => {
+                    if (result.isValid) {
+                        if (config.showOwnMessages) {
+                            ui.setContent('input-container.message', '');
+                        }
+                    } else if (result.diagnosis === 'required-missing') {
+                        // nothing??
+                    } else {
+                        if (config.showOwnMessages) {
```

Comment:
This honestly should probably never be run - how do you select an error-triggering option from a predefined set of options? It was here already, though, so maybe there's some magic way it could happen?